tools/loader/ImageClassifier.cpp

```diff
@@ -245,10 +254,30 @@ static void printTopKPairs(const std::vector<FloatIndexPair> &topKPairs) {
       llvm::outs() << "\t\t\t\t\t";
     }
     llvm::outs() << "\tLabel-K" << i + 1 << ": " << label << " (probability: "
-                 << llvm::format("%0.4f", topKPairs[i].first) << ")\n";
+                 << llvm::format("%0.4f", topKPairs[i].first) << ")"
+                 << " in the top " << topKPairs.size() << " pairs\n";
```

Comment:
nit: I'd prefer leaving this print message as it was before. It already prints `topKPairs.size()` times anyway.